yarn-error.log

```diff
@@ -0,0 +1,5734 @@
+Arguments: 
```

Comment:
let's not commit this file

Reply:
Totally looked over that file, will remove it